src/Select.scss

```diff
@@ -0,0 +1,8 @@
+.select-option {
+  button {
```

Comment:
There's no button inside the `select-option`, so this doesn't seem to be used right now, right?